waiter/config-full.edn

```diff
@@ -350,6 +350,11 @@
                                 "scale-factor" 1
                                 "scale-up-factor" 0.1}
 
+ ;; Service description parameters with numeric values, e.g. cpus, can be configured to have
+ ;; maximum value limits. When not configured, these defaults will be used:
+ :service-description-resource-limits {"cpus" 100
```

Comment:
Would `:service-description-maximums` be a better name?

Reply:
Renamed to `service-description-upper-limits`. Okay with the name?